### ISSUE TITLE
I visualize the img2 in max_ssim.py, but it is not the gif shown in the home page.  what mistake do I make? The following is the code.

### ISSUE THREAD

**Issue by THHHomas** [Po-Hsun-Su/pytorch-ssim#21]
import pytorch_ssim
import torch
from torch.autograd import Variable
from torch import optim
import cv2
import numpy as np

npImg1 = cv2.imread("einstein.png")

img1 = torch.from_numpy(np.rollaxis(npImg1, 2)).float().unsqueeze(0)/255.0
img2 = torch.rand(img1.size())

if torch.cuda.is_available():
    img1 = img1.cuda()
    img2 = img2.cuda()

img1 = Variable( img1,  requires_grad=False)
img2 = Variable( img2, requires_grad = True)

# Functional: pytorch_ssim.ssim(img1, img2, window_size = 11, size_average = True)
print(img1.shape,img2.shape)
ssim_value = pytorch_ssim.ssim(img1, img2).item()
print("Initial ssim:", ssim_value)

# Module: pytorch_ssim.SSIM(window_size = 11, size_average = True)
ssim_loss = pytorch_ssim.SSIM()

optimizer = optim.Adam([img2], lr=0.01)

while ssim_value < 0.95:
    optimizer.zero_grad()
    ssim_out = -ssim_loss(img1, img2)
    ssim_value = - ssim_out.item()
    print(ssim_value)
    ssim_out.backward()
    optimizer.step()
    ii = np.transpose(img2[0].detach().cpu().numpy(), (1,2,0))
    cv2.imshow("pre", ii*255)
    print(np.max(ii*255))
    cv2.waitKey(100)
